pyproject.toml

```diff
@@ -132,13 +132,7 @@ omit = [
 show_missing = true
 
 [tool.semantic_release]
-version_toml = "pyproject.toml:tool.poetry.version"
-branch = "main"
-upload_to_repository = true
-upload_to_release = true
-build_command = "poetry build"
-commit_subject = "chore(release): v{version}"
-commit_author = "github-actions <github-actions@github.com>"
+version_source = "tag"
```

Comment:
Couldn't we keep the "version_toml" bit? Or is this only used for writing?

Reply:
That would conflict with the `version_source` since only one `version_` config can be set..